README.md

```diff
@@ -1,16 +1,34 @@
-# Title
+# Ruby Basics Lab
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+* Define the body of each method
 
 ## Introduction
 
-## SWBAT 1
+For this lab, you're going to be reviewing the basics of the Ruby language. Once
+you fork and clone the lab down onto your computer, run the test suite with the
+`learn` or `rspec` command. Read the test output in your terminal to get your
+tests to pass.
 
-## SWBAT 2
+## Define the Body of Each Method
+
+We've already defined the methods for you––you need to write the code inside of
+them to give them the desired behavior, as defined by the tests. Refer back to
+the earlier lessons on reading test output for help.
+
+* `#division`: It returns quotient of the first number divided by the second
+  number
+* `#assign_variable`: It takes an argument of a persons name and assign it to a
```

Comment:
assigns

Reply:
How are we going to test this? Based on the signature below we're going to pass in "Godzilla" and then define a local var `'name`. How will we know the local `name` (now out of scope) is `== Godzilla` ?